### PR TITLE
Refactor natspec

### PR DESCRIPTION
**Pull request by lbeder**
- [x] Remove @param and @return values
- [x] Add explicit "Requirements" section for security and other non-trivial assumptions

### CODE REVIEW

File: contracts/IConversionPathFinder.sol [hunk 1]
```diff
@@ -3,9 +3,9 @@ pragma solidity 0.6.12;
 
 import "./token/interfaces/IReserveToken.sol";
 
-/*
-    Conversion Path Finder interface
-*/
+/**
+ * @dev Conversion Path Finder interface
```
Comment (yudilevi) [inline contracts/IConversionPathFinder.sol:7]: the reason we didn't add @dev before was that it wasn't supported by gitbooks - if I'm not mistaken.
so we just need to make sure that one works.

File: contracts/NetworkSettings.sol [hunk 2]
```diff
@@ -91,12 +76,14 @@ contract NetworkSettings is INetworkSettings, Owned, Utils {
 
     /**
      * @dev sets the network fee
-     * can be executed only by the owner
      *
-     * @param newNetworkFee new network fee in ppm units
+     * Requirements:
```
Comment (yudilevi) [inline contracts/NetworkSettings.sol:80]: we need to remember to point out that these are in PPM (and later on rename it to feePPM)

**Review by yudilevi — APPROVED** [review]

File: contracts/IConversionPathFinder.sol (hunk 1 above)
Comment (lbeder) [reply contracts/IConversionPathFinder.sol:7]: Ah, super weird. It's pretty common. In any case, it'd be easy to revert later on

File: contracts/NetworkSettings.sol (hunk 2 above)
Comment (lbeder) [reply contracts/NetworkSettings.sol:80]: I'll add a comment

Comment (yudilevi) [reply contracts/NetworkSettings.sol:80]: cool

**Pull request merged**